tests/scripts/check-generated-files.sh

```diff
@@ -22,3 +22,4 @@ check()
 
 check library/error.c scripts/generate_errors.pl
 check library/version_features.c scripts/generate_features.pl
+check visualc/VS2010/mbedTLS.vcxproj scripts/generate_visualc_files.pl
```

Comment:
What about the other files generated by this script?

Reply:
@mpg: Sorry, you are right, I did not look carefully enough. I will modify the script tomorrow to also check the other visual studio files.